src/goPackages.ts

```diff
@@ -81,13 +80,23 @@ function gopkgs(workDir?: string): Promise<Map<string, string>> {
 				});
 				return resolve(pkgs);
 			}
-
+			const stdPkgs = new Map<string, string>();
+			const otherPkg = new Map<string, string>();
 			output.split('\n').forEach((pkgDetail) => {
 				if (!pkgDetail || !pkgDetail.trim() || pkgDetail.indexOf(';') === -1) return;
-				const [pkgName, pkgPath] = pkgDetail.trim().split(';');
-				pkgs.set(pkgPath, pkgName);
+				const [pkgName, pkgPath, pkgDir] = pkgDetail.trim().split(';');
+				if (pkgDir.includes(process.env['GOROOT'])) {
+					stdPkgs.set(pkgPath, pkgName);
+				} else {
+					otherPkg.set(pkgPath, pkgName);
+				}
 			});
-
+			Array.from(stdPkgs.keys()).forEach((val)=>{
+				pkgs.set(val, stdPkgs.get(val));
+			})
+			Array.from(otherPkg.keys()).forEach((val)=>{
+				pkgs.set(val, otherPkg.get(val));
+			})
```

Comment:
@ramya-rao-a This looks very crude to me :) esp maintaining two maps and merging them. Could you suggest a way to improve it? Thanks!

Reply:
Instead of maintaining 2 different maps, we can go back to the single map as before.
That map uses string -> string at the moment
We can change that to string -> PackageInfo where PackageInfo is an interface
```
export interface PackageInfo {
   name string;
   isStd boolean;
}
```

Reply:
This makes it extendable such that in the future if we want to extract more info from the `gopkgs` tool, we can do so.